elide-model-config/src/main/java/com/yahoo/elide/modelconfig/validator/DynamicConfigValidator.java

```diff
@@ -348,6 +365,12 @@ private String getInheritedTable(Table table, String tableName) {
         return getInheritedAttribute(action, tableName);
     }
 
+    private List<Argument> getInheritedArguments(Table table, List<Argument> arguments) {
+        Inheritance action = () -> table.getArguments();
```

Comment:
method reference is preferable to lambda.
```suggestion
        Inheritance action = table::getArguments;
```